src/pages/[productSlug]/integrations/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { ProductSlug } from 'types/products'
+import { cachedGetProductData } from 'lib/get-product-data'
+import ProductIntegrationsLanding from 'views/product-integrations-landing'
+
+export async function getStaticPaths() {
+	const paths = ['vault', 'packer'].map((productSlug: ProductSlug) => ({
+		params: { productSlug },
+	}))
+	console.log('getStaticPaths', paths)
+	return {
+		paths,
+		fallback: true,
```

Comment:
💭 https://nextjs.org/docs/api-reference/data-fetching/get-static-paths#fallback-pages

```suggestion
		fallback: false,
```

Reply:
:brycefax: